src/scenes/ModulesRouter/ModuleEntry.tsx

```diff
@@ -2,32 +2,41 @@ import { Box, Grid, Typography, useTheme } from '@mui/material';
 
 import Button from 'src/components/common/button/Button';
 import useNavigateTo from 'src/hooks/useNavigateTo';
+import { useLocalization } from 'src/providers';
 import strings from 'src/strings';
 import { Module } from 'src/types/Module';
+import { getDateRangeString } from 'src/utils/dateFormatter';
 
 interface ModuleEntryProps {
+  index: number;
+  last?: boolean;
```

Comment:
should we use a last-child selector instead of passing in `last` as a prop?

Reply:
Oh yeah I'm so used to not having selectors like that in mobile apps that I forget to use them sometimes, good call 👍